docker-compose.dev.yml

```diff
@@ -1,27 +1,27 @@
 version: "3.1"
 
 services:
-  db:
+  db_dev:
     image: postgres
     restart: always
     environment:
       POSTGRES_PASSWORD: trytofindme
     volumes:
-      - ./db:/var/lib/postgresql/data
+      - ./db:/var/lib/postgresql/data_dev
```

Comment:
It is not working

DATA_IN_LOCAL_DIRECTORY:DATA_IN_CONTAINER

Postrges not read data from custom directory "data_dev"

Reply:
Will it work if I make it so?
-./db:/var/lib/postgresql/db_dev/data

Reply:
NO
You need to change your local path not in container

Reply:
-./db/db_dev:/var/lib/postgresq/data?

Reply:
Yes)